extra/Android.mk

```diff
@@ -526,3 +547,78 @@ LOCAL_MODULE_CLASS := ETC
 LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/permissions
 LOCAL_SRC_FILES := $(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
+
+# override lineageOSnoLOSEmail
+include $(CLEAR_VARS)
+LOCAL_MODULE := noLOSEmail
+LOCAL_SRC_FILES := empty.apk
+LOCAL_CERTIFICATE := PRESIGNED
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := APPS
+LOCAL_DEX_PREOPT := false
+LOCAL_MODULE_SUFFIX := .apk
+LOCAL_OVERRIDES_PACKAGES := Email
+include $(BUILD_PREBUILT)
+
+# override lineageOS Messaging
+include $(CLEAR_VARS)
+LOCAL_MODULE := noLOSMessaging
+LOCAL_SRC_FILES := empty.apk
+LOCAL_CERTIFICATE := PRESIGNED
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := APPS
+LOCAL_DEX_PREOPT := false
+LOCAL_MODULE_SUFFIX := .apk
+LOCAL_OVERRIDES_PACKAGES := messaging
+include $(BUILD_PREBUILT)
+
+# override lineageOS Snap
+include $(CLEAR_VARS)
+LOCAL_MODULE := noLOSSnap
+LOCAL_SRC_FILES := empty.apk
+LOCAL_CERTIFICATE := PRESIGNED
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := APPS
+LOCAL_DEX_PREOPT := false
+LOCAL_MODULE_SUFFIX := .apk
+LOCAL_OVERRIDES_PACKAGES := Snap
+include $(BUILD_PREBUILT)
+
+# override lineageOS Camera2
+include $(CLEAR_VARS)
+LOCAL_MODULE := noLOSSnap
```

Comment:
is that a typo or should it really be `LOCAL_MODULE := noLOSSnap` here like in the above block? If that is not a typo: won't that cause conflicts? I would assume more something like `noLOScamera2` here tbh..

Reply:
Yes - it should be`noLOScamera2`

Reply:
Commited the change